tutorials/fit_convergence.rst

```diff
@@ -0,0 +1,41 @@
+.. |br| raw:: html
+
+    <br>
+
+Testing a fit's convergence
+===========================
+If you'd like an in-depth look at how well a fit has converged,
+the fastest way is to run a fit from the terminal
+(see the `Quickstart <quickstart.rst>`_)
+and set `diag_plot=True` in your *.json* parameter file.
+This produces a diagnostic figure that, using the fit to the DSHARP observations
```

Comment:
Why is this file not in the docs directory? (Same for the other rst files in tutorials)

Reply:
I'm now classing tutorials as a mix of jupyter tutorials and .rst tutorials. I think it's best to keep these together, as those that can't be run as a notebook are still tutorials that explain how to examine an aspect of the fit. They're in `tutorials` instead of `docs` because you wanted to keep `tutorials` in a directory above `docs`. I'd rather move everything into `docs` than separate the .rst tutorials from the .ipynb tutorials, but I assume you'd prefer the opposite? ha

Reply:
I guess it doesn't matter. I'd prefer docs as a single unified location, I guess

Reply:
Ok cool, then would you prefer to keep them in `frank/tutorials` (the current arrangement), or move `frank/tutorials` --> `frank/docs/tutorials`?

Reply:
Move them? You'll need to update the links and remove the copy from the circle CI build

Reply:
54a5ae2 and 69e415b do this. I tested rebuilding the docs after these changes with `make clean` then `make html` and it all looks ok, but do let me know if you can think of anything I missed between those 2 commits.

Reply:
Looks fine, please remove the commented lines.